.editorconfig

```diff
@@ -0,0 +1,11 @@
+# Editor configuration, see https://editorconfig.org
```

Comment:
We currently also have a second .editorconfig file in the /axolotl-web subfolder. The two files are identical.

I suggest to remove the file from `axolotl-web/.editorconfig` - as to avoid duplication. 

> When opening a file, EditorConfig plugins look for a file named .editorconfig in the directory of the opened file and in every parent directory. A search for .editorconfig files will stop if the root filepath is reached or an EditorConfig file with root=true is found.

https://editorconfig.org/#file-location

Reply:
I am new to this editor-config files, the reason why i actually added one in the sub-folder was, because the vscode prettier doesn't work well if the package.json is not in the root folder. So i opened the project again in the subfolder.

Reply:
But we can remove it again, i think i have resolved it now

Reply:
I use IntelliJ Webstorm: if VSCode needs it, lets definitely keep it.